src/features/advanced/inv-compress-inventory-info.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  applyScopedClassCssRule,
+  applyScopedCssRule,
+} from '@src/infrastructure/prun-ui/refined-prun-css';
+import classes from './inv-compress-inventory-info.module.css';
+import css from '@src/utils/css-utils.module.css';
+import ContextControls from '@src/components/ContextControls.vue';
+import { getInvStore } from '@src/core/store-id';
+import { sitesStore } from '@src/infrastructure/prun-api/data/sites';
+import {
+  getEntityNaturalIdFromAddress,
+  getLocationLineFromAddress,
+} from '@src/infrastructure/prun-api/data/addresses';
+import { warehousesStore } from '@src/infrastructure/prun-api/data/warehouses';
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+
+async function onTileReady(tile: PrunTile) {
+  if (!tile.parameter) {
+    return;
+  }
+  await $(tile.anchor, C.StoreView.container);
+
+  const store = getInvStore(tile.parameter);
+  if (!store) {
```

Comment:
I don't see why we need this `if`. Every place we access the store, we do it via `store?.`. And if, for some reason, no store was found, we still probably want to display `ContextControls` with `INV`. Otherwise, we risk a broken layout because we apply absolute positioning to`C.StoreView.row`.

One place where you will need to change is `switch (store.type) {` to `switch (store?.type) {`